modules/network/main.tf

```diff
@@ -78,21 +120,6 @@ resource "azurerm_network_security_rule" "sg_weka_ui" {
   network_security_group_name = azurerm_network_security_group.sg[0].name
 }
 
-resource "azurerm_network_security_rule" "sg_deny_outbound_internet" {
```

Comment:
I agree that we shouldn't use it
How will you test private network ?
You will put this rule manually ?